turbinia/config/turbinia_config.py

```diff
@@ -16,20 +16,57 @@
 from __future__ import unicode_literals
 
 
-"""Dummy Turbinia config file."""
+"""Turbinia Config Template"""
 
-# Turbinia Config
+##########
+# COMMON #
+##########
 
-# 'PSQ' is currently the only valid option
+# Which user account Turbinia runs as
+USER = 'turbinia'
+
+# Turbinia's installion directory
+TURBINIA_DIR = '/opt/turbinia'
+
+# Turbinia source code directory (local git repository)
+SRC_DIR = '%s/src' % TURBINIA_DIR
+
+# GCSFuse mount for Turbinia scripts & logging and Google Cloud Functions
+MOUNT_POINT = '/mnt/turbinia'
+
+# Turbinia CLI
+TURBINIACTL = '%s/turbinia/turbiniactl' % SRC_DIR
+
+# Virtualenv directory
+TURBINIAENV = '%s/turbinia-env' % HOME_DIR
+
+# Virtualenv activator
+VIRTUALENV_ACTIVATE = '%s/bin/activate' % TURBINIAENV
+
+# GCS bucket that has Turbinia-specific scripts and
+# that can be used to store logs.
+BUCKET = 'turbinia'
+
+# Local directory for temporary data
+TMP_DIR = '/var/tmp'
+
+############
+# TURBINIA #
```

Comment:
Really this whole file is for Turbinia.  Aren't these really still part of the common config?  They are used when running on cloud or locally.  Maybe either remove it, or mark it as "Turbinia Server".  Another option would be to move some of the common things like LOG_FILE and OUTPUT_DIR above, and then mark this section as "Turbinia Internals".

Reply:
Replaced COMMON with TURBINIA.